spec/std/log/broadcast_backend_spec.cr

```diff
@@ -0,0 +1,63 @@
+require "spec"
+require "log/config"
+require "../../support/log"
+
+private def s(value : Log::Severity)
+  value
+end
```

Comment:
Why is this needed? Passing `:info` or `:error` directly without this indirection works just fine.

Same for all other specs.

Reply:
In `exp.should eq(x)` the `x` is not autocasted.
And in the config specs since there are hash of arrays of config, the autocasting does not apply that either.

I agree there are many of this invocations that are not needed, but when working in bulk it was easier for me this way.

Reply:
Why not `private alias S = Log::Severity`?

Reply:
That would mean `S.new(value)` rather than `s(value)` doesn't it?

Reply:
Makes sense. It's a good workaround :-)

Reply:
The alias or how it is right now?

Reply:
How it is right now